CMakeLists.txt

```diff
@@ -263,6 +264,10 @@ set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
 
 configure_file("config.h.in" "config.h")
 add_definitions(-DHAVE_CONFIG_H)
+if (MSVC)
+	add_definitions(-D_CRT_SECURE_NO_WARNINGS)
+	add_definitions(-D_CRT_NONSTDC_NO_WARNINGS)
```

Comment:
are you sure about this? aren't both of these valuable warnings?

Reply:
I am, I'll add a comment to there.